releasenotes/notes/24263.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - https://github.com/istio/istio/issues/18487
```

Comment:
Second time I've seen this pattern today, so adding some clarification. There are two different ways of referencing PRs. Number only implies the Istio repo. Having a URL directs to the actual PR. Both work.

Reply:
Thanks for the clarification @brian-avery .

I have added 2 things here. Let me know if this is right?
1. The actual issue reported https://github.com/istio/istio/issues/18487
2. PR to resolve that issue 24263

I think resolved PR should be enough to be included because it will have an issue link mentioned?

Reply:
This should only reference issues resolved in this PR.

End result will be that it generates a line like: 

Improved the sidecar injection control based on revision at a per-pod level (Issue 24801)

Reply:
Perfect. Thanks